server/views/applications/taskList.njk

```diff
@@ -42,24 +42,6 @@
           {% include '../partials/taskListItems.njk' %}
 
           <li>
-            <h2 class="app-task-list__section">
-              Submit your application
-            </h2>
-
-            {{
-              govukCheckboxes({
```

Comment:
I think we can remove the `govukCheckboxes` import from this file too.

Reply:
ah good spot. I foolishly put on auto-merge for this branch, will do a small PR to remove this :)